client/src/components/map/layers/networks-layer/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { useMemo } from 'react';
+
+import { GeoJSONSourceSpecification } from 'maplibre-gl';
+import { Source, Layer, SymbolLayer } from 'react-map-gl/maplibre';
+
+import { LayerProps } from '@/types/layers';
+
+import { useMapNetworks } from '@/hooks/networks';
+
+export type NetworksLayerProps = LayerProps & {
+  beforeId?: string;
+};
+
+const getClusterSource = (source: GeoJSONSourceSpecification): GeoJSONSourceSpecification => ({
+  type: 'geojson',
+  data: source.data,
+  cluster: true,
+  clusterMaxZoom: 14, // Max zoom to cluster points on
+  clusterRadius: 100, // Radius of each cluster when clustering points
+  clusterProperties: {
+    organizationsCount: ['+', ['get', 'organizationsCount']],
+    projectsCount: ['+', ['get', 'projectsCount']],
+  },
+});
+
+const sharedLayout: SymbolLayer['layout'] = {
+  'icon-image': 'rect',
+  'icon-allow-overlap': true,
+  'icon-ignore-placement': true,
+  'text-size': 12,
+  'text-anchor': 'top',
+  'text-allow-overlap': true,
+  'text-ignore-placement': true,
+};
+
+const organizationsStyle: { layout: SymbolLayer['layout']; paint: SymbolLayer['paint'] } = {
+  layout: {
+    ...sharedLayout,
+    'icon-size': ['step', ['get', 'organizationsCount'], 1, 1, 1.5, 10, 2, 50, 2.5, 100, 2.5],
+    'icon-offset': [10, 0],
+    'text-field': ['number-format', ['get', 'organizationsCount'], { locale: 'en' }],
```

Comment:
We don't know exactly how many organisations/projects we're gonna have in one place and it's possible we reach thousands. For this reason, could you format the number as in Scientific Evidence?

https://github.com/Orcasa-Platform/orcasa-review4c/blob/main/javascript/map/map-layers.js#L59-L90